packages/labs/cli/src/lib/init/element-starter/templates/lib/element.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+import {
+  javascript,
+  kabobToPascalCase,
+} from '@lit-labs/gen-utils/lib/str-utils.js';
+import {Language} from '../../../../commands/init.js';
+
+export const generateElement = (
+  elementName: string,
+  language: Language
+): FileTree => {
+  const directory = language === 'js' ? 'lib' : 'src';
+  return {
+    [directory]: {
+      [`${elementName}.${language}`]:
+        language === 'js'
+          ? generateTemplate(elementName, language)
+          : generateTemplate(elementName, language),
+    },
+  };
+};
+
+const generateTemplate = (elementName: string, lang: Language) => {
+  const className = kabobToPascalCase(elementName);
+  return javascript`import {LitElement, html, css} from 'lit';${
+    lang === 'js'
+      ? ''
+      : `
+import {property, customElement} from 'lit/decorators.js';`
+  }
+
+/**
+ * An example element.
+ *
+ * @fires count-changed - Indicates when the count changes
+ * @slot - This element has a slot
+ * @csspart button - The button
+ * @cssprop --${elementName}-font-size - The button's font size
+ */${
+   lang === 'js'
+     ? ''
+     : `
+@customElement('${elementName}')`
+ }
+export class ${className} extends LitElement {
+  static ${lang === 'js' ? '' : 'override '}styles = css\`
```

Comment:
I'm really skeptical of the `override` addition here. Seems like most TypeScript devs don't use this and it adds extra noise. We got questions about this on our GH discussions recently. @kevinpschaaf ?